### PR TITLE
feat: clockface upgrade (3.0.0 beta)

### PR DESCRIPTION
**Pull request by daviddkkim**
This is the master PR that will introduce the clockface 3.0.0 beta change. This is an ongoing work, multiple PR-s will first merge into this before the big PR merges into the `master`.

### CODE REVIEW

**Comment by genehynson** [comment]
@daviddkkim  can we switch this to a draft PR so it isn't accidentally merged? ("convert to draft" in the top right) Also it won't send emails to everyone on the UI team if it's still in "draft" stage 😄

**Comment by daviddkkim** [comment]
> @daviddkkim can we switch this to a draft PR so it isn't accidentally merged? ("convert to draft" in the top right) Also it won't send emails to everyone on the UI team if it's still in "draft" stage 😄

Yes! Sorry about the notifications

**Comment by ui-ci-bot-influx** [comment]
Metric | Score | Level
------------ | ------------- | ------------- 
performance | 60 | :red_circle:
accessibility | 96 | :green_circle:
seo | 90 | :green_circle:
pwa | null | :hankey:
best-practices | 100 | :green_circle:


[Compare your scores with master branch](https://influxdb.aws.influxdata.io/orgs/844910ece80be8bc/dashboards/073c4b95f1089000?lower=now%28%29%20-%202d&vars%5Blighthouse_score_branch%5D=2332)

File: src/authorizations/components/redesigned/ResourceAccordionHeader.tsx [hunk 1]
```diff
@@ -65,9 +65,6 @@ export const ResourceAccordionHeader: FC<OwnProps> = props => {
         stretchToFitWidth={true}
         alignItems={AlignItems.Center}
         style={{textAlign: 'start'}}
-        onClick={(e: any) => {
```
Comment (ChitlangeSahas) [inline src/authorizations/components/redesigned/ResourceAccordionHeader.tsx:68]: The `FlexBox` does not have `onClick` anymore, does this sound right @daviddkkim?

**Comment by ui-ci-bot-influx** [comment]
Metric | Score | Level
------------ | ------------- | ------------- 
performance | 55 | :red_circle:
accessibility | 95 | :green_circle:
seo | 90 | :green_circle:
pwa | null | :hankey:
best-practices | 100 | :green_circle:


[Compare your scores with master branch](https://influxdb.aws.influxdata.io/orgs/844910ece80be8bc/dashboards/073c4b95f1089000?lower=now%28%29%20-%202d&vars%5Blighthouse_score_branch%5D=2332)

File: src/authorizations/components/redesigned/ResourceAccordionHeader.tsx (hunk 1 above)
Comment (daviddkkim) [reply src/authorizations/components/redesigned/ResourceAccordionHeader.tsx:68]: The onclick prop addition to flexbox happened to clockface 2.0 after we created clockface 3.0. So 3.0 will have it as well!

Comment (daviddkkim) [reply src/authorizations/components/redesigned/ResourceAccordionHeader.tsx:68]: I merged the newest clockface updates to the beta version and published a new version in `3.0.0-beta.3`

Comment (ChitlangeSahas) [reply src/authorizations/components/redesigned/ResourceAccordionHeader.tsx:68]: Got you! So I will add this back after pulling in the latest `clockface`

**Comment by genehynson** [comment]
@daviddkkim @ChitlangeSahas can we close this now?

**Comment by daviddkkim** [comment]
done

**Pull request closed without merging**